src/controllers/enterpriseController.test.ts

```
@@ -1,151 +1,442 @@
+        query: { tolerance: "invalid" },
+      } as unknown as AuthRequest;
+
+      await getTreasury(req, res, makeNext());
+
+      expect(mockTreasuryService.getEnterpriseTreasury).toHaveBeenCalledWith(
+        "org-123",
+        0.01, // default
+      );
+    });
+
+    it("clamps tolerance value between 0 and 100", async () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)